bigml/ensemble.php

```diff
@@ -101,7 +105,7 @@ public function __construct($ensemble, $api=null, $max_models=null, $storage="st
             $this->boosting = isset($ensemble->object->boosting);
 
             if ($this->boosting) {
-                $this->boostedensemble = new BoostedEnsemble($ensemble, $this->api, $max_models, $storage);
+                $this->boostedensemble = new BoostedEnsemble($ensemble, $api=null, $max_models=null, $storage="storage");
```

Comment:
This syntax kind of puzzles me. It looks like you are trying to use some kind of "named parameters" when calling the BoostedEnsemble constructor. Though, I cannot find any hints in PHP docs that named parameters are available -- in any version of PHP (and even if they were in latest PHP, I think we should also consider backward compatibility with previous PHP versions).

So, as far as I understand, the above code just overrides a bunch of local variables (which are furthermore input parameters of that function) and pass their values to the constructor.

On the one hand, I do not think this "argument overriding" could be considered good practice, since in some future extension of the code, it would not be evident enough that the input args were overridden in the call to BoostedEnsemble.

On the other hand, I wonder why it is required to hardcode those parameters which are "pass-through" (meaning, they are specified by the callee) and additionally they have exactly the default values that you are hardcoding...

Maybe there is something that I am missing here...

Reply:
This looks like some bleed-through from a previous commit. The new change is incorrect code I fixed in the previous PR. I'll resubmit the correct line.